tools/wptrunner/wptrunner/executors/executormarionette.py

```diff
@@ -916,6 +979,9 @@ def run_test(self, test):
                                                               "timeout": timeout,
                                                               "width": 800,
                                                               "height": 600})["value"]
+        for item in rv["extra"].get("reftest_screenshots", []):
```

Comment:
Again, same comment re wrapping a single-item list vs whoever is accumulating a list handling it.